src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -13,7 +13,9 @@ type ToggleSettingOptionRowProps = {
     /** Icon to be shown for the option */
```

Comment:
inconsistent whitespaces (one new line before the last one). Maybe add them before every comment?